core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -155,11 +157,23 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
 
   .oppia-exploration-footer .oppia-exploration-footer-sharing-links ul {
     display: block;
-    float: left;
+    float: right;
     margin: -8px;
   }
 
   .oppia-exploration-footer .oppia-exploration-footer-sharing-links li {
     float: left;
   }
+
+  @media (max-width: 420px) {
+    .oppia-exploration-footer-sharing-links {
+      display: none;
+    }
+  }
+
+  @media (max-width:641px) {
+    .oppia-hide-mobile {
```

Comment:
Just use share-exploration-footer (maybe rename throughout to oppia-share-exploration-footer). Otherwise this name is pretty generic.